src/Endpoint/test/VMware.ScriptRuntimeService.APIGateway.Tests/ScriptExecutionFileStorageTests.cs

```diff
@@ -90,9 +87,9 @@ public class ScriptExecutionFileStorageTests {
             completedScripts.Add(args.ScriptId);
          };
 
-         _scriptExecutionFileStorage.StartStoringScriptExecution(_userId1, _runspace, _scriptId1, _scriptName1);
-         _scriptExecutionFileStorage.StartStoringScriptExecution(_userId2, _runspace, _scriptId2, _scriptName2);
-         _scriptExecutionFileStorage.StartStoringScriptExecution(_userId2, _runspace, _scriptId3, _scriptName3);
+         _scriptExecutionFileStorage.StartStoringScriptExecution(_userId1, _runspace, _scriptId1, _scriptName1, false);
```

Comment:
Shouldn't there be unit tests when the parameter is `true`?

Reply:
I've added a test